app/services/backend_service.rb

```diff
@@ -0,0 +1,14 @@
+class BackendService
+  def self.conn
+    Faraday.new(url: ENV["BACKEND_URL"])
+  end
+
+  def self.flights_search(query)
+    response = conn.get('/flight_search') do |req|
```

Comment:
Currently our search route in BE is `/api/v1/search` so I think we'll want to change the `/flight_search' path here.

Reply:
This has been updated.